website/src/controllers/payments/paypalBillingSetup.js

```diff
@@ -1,3 +1,5 @@
+require('babel-core/register');
```

Comment:
You should not use babel-core/register. When exactly this code will have to run from the command line? If only locally you can use https://babeljs.io/docs/usage/cli/